pkg/agent/openflow/pipeline.go

```diff
@@ -1458,7 +1458,7 @@ func (c *client) gatewayARPSpoofGuardFlows(gatewayIP net.IP, gatewayMAC net.Hard
 		flows = append(flows, SpoofGuardTable.BuildFlow(priorityNormal).MatchProtocol(binding.ProtocolARP).
 			MatchInPort(config.HostGatewayOFPort).
 			MatchARPSha(gatewayMAC).
-			MatchARPSpa(c.nodeConfig.NodeIPv4Addr.IP).
+			MatchARPSpa(c.nodeConfig.NodeTransportIPv4Addr.IP).
```

Comment:
Trying to understand why is this required and why this will be NodeTransportIP.
@gran-vmv I assume this is for external -> NodePort -> AntreaIPAM Pod, and the Node's IP should be in the same subnet as AntreaIPAM Pod, then the transport interface should be the one it sends out the ARP request, why the flow matches hostGateway port?

Reply:
As above comment said:
```
		// When the uplink bridged and NodePort/HostPort is accessed, Node will send ARP request to HostGatewayOFPort
		// with arp_spa=NodeIP. This flow is to accept these ARP requests.
```
The destination is Per-Node IPAM Pod and source is external/AntreaIPAM Pod.
I still not sure why this occur only when NodePort/HostPort accessed. The ARPSpa is correct when the we access local HostPort from Node.